### ISSUE TITLE
403 on requesting Public API  ​/v3​/vaccination​/generateOTP

### ISSUE THREAD

**Issue by rvshah512** [cowinapi/developer.cowin#538]
It seems the Public API page was updated today and the Vaccination Verification API's are not working. On requesting 
​/v3​/vaccination​/generateOTP a response of 403 {message: Forbidden} is received.

**Comment by paramhq** [comment]
![image](https://user-images.githubusercontent.com/3767382/133729270-cd384508-666a-4bfa-89bf-92ac374ef036.png)

It now shows as Protectd. What change should we do? Any updates?

**Comment by rvshah512** [comment]
@sukhdev22 Yes, they changed it to a protected API but it is still shown in the Public API page. I guess one needs to apply for getting access to the production API key just like the other Co-WIN protected APIs.

**Comment by ankit8588** [comment]
@rvshah512 Yes you are right. But the emails to partner address is bouncing back. Looks like some issue, even the sample is no more sending the OTP despite success message.

Cowin Team - Kindly help.

**Comment by ankit8588** [comment]
any update to anyone..

the apis are working using api key. Is there a way no procure and apply for api key. `i have dropped emails and reminders.

**Comment by rvshah512** [comment]
@ankit8588 We have applied for ASP registration. No response yet.

**Comment by ankit8588** [comment]
Thanks. As per the info on the main page. We just have to send an email to the given address.

Is there a form as well for the same?

**Comment by rvshah512** [comment]
The said API's have been removed from the website.

**Comment by ankit8588** [comment]
No, it has been moved to protected section now.

We need APIKey. Did you get an access to it? How much time do they take on average to approve the API key for protected API? Any idea.

**Comment by rvshah512** [comment]
@ankit8588 It's been a while since we applied. Haven't received the key yet.

**Comment by praveen-gm** [comment]
How does one test this API on the sandbox server? there is an API key but there are no samples provided

**Comment by ankit8588** [comment]
The curl commands are there and it is quite straightforward. Just code a
rest call and it works like a charm in a Sandbox environment.

I am still waiting for the Production API Key.

Thanks,

**Comment by praveen-gm** [comment]
Thanks for the reply,

{
  "mobile_number": "9898989999",
  "full_name": "Kritika Pani"
}

this is the sample provided on the site, but this doesn't work.
i got response {"errorCode":"USRAUT0015", "error":"Your account does not exist"}

Are there any sample users that I can check the complete flow of authenticating and getting the vaccination status?

**Comment by ankit8588** [comment]
This is an expected response and you will get a valid response once you
pass on the actual credentials of the test environment. I do not have them
but earlier even these credentials were working fine.

**Comment by ankit8588** [comment]
Hey, anyone got the API key for production environment?

**Comment by rvshah512** [comment]
@ankit8588 Not yet. What about you?

**Comment by rahulkathet** [comment]
@ankit8588  not yet
